packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -24,29 +80,100 @@
   "form.attribute.item.number.type.decimal": "decimal (ex: 2.22)",
   "form.attribute.item.number.type.float": "flotante (ex: 3.3333333333)",
   "form.attribute.item.number.type.integer": "entero (ej: 10)",
+  "form.attribute.item.privateField": "Campo privado",
+  "form.attribute.item.privateField.description": "Este campo no aparecerpa en la respuesta de la API",
```

Comment:
typo: "aparecerá"

Reply:
"Este campo no se mostrara en la respuesta de la API"...... "aparecerá" its not a good term for this case